### PR TITLE
Create draft schedule for discussion.  Add CJW as Test Infra shadow.

### PR DESCRIPTION
**Pull request by jberkus**
This is a DRAFT schedule for 1.11.  I'd like to finalize it by Monday.

I haven't changed the essential time relationships of the release weeks from 1.10.  However, there are some concerns:

1. the American July 4th holiday comes right after the release.  This is a potential source of problems in that (a) it makes for a weird start for 1.12 and (b) if the release slips by more than a couple days we may need to shift it by over a week.

2. A few deadlines happen during KubeCon.EU.  Do these need to be moved?

### CODE REVIEW

**Comment by jberkus** [comment]
@calebamiles @idvoretskyi @jdumars @AishSundar @BenTheElder @kbarnard10

File: releases/release-1.11/release-1.11.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+* [Release Team](http://bit.ly/kube111-team)
+* [Zoom](http://bit.ly/kube111-zoom)
+* [Slack](https://kubernetes.slack.com/messages/sig-release/)
+* [Forum](https://groups.google.com/forum/#!forum/kubernetes-sig-release)
+* [Feature Tracking Sheet]()
+* [Milestone Process](https://github.com/kubernetes/community/blob/master/contributors/devel/release/issues.md)
+* [Burndown / Meeting Minutes](http://bit.ly/kube111-burndown)
+* [Retrospective Document](http://bit.ly/kube111retro)
+
+**tl;dr** The 1.10 release cycle begins on **Tuesday, April 3rd, 2018**, and ends on release day, **Wednesday, June 27th**.  Feature freeze is **Tuesday, April 24th**.  Code freeze begins **Monday, May 28th** and ends **Wednesday, June 20th**. Docs must be completed and reviewed by **Friday, May 11th**.
```
Comment (BenTheElder) [inline releases/release-1.11/release-1.11.md]: 1.11? also a few other places

Comment (jberkus) [reply releases/release-1.11/release-1.11.md]: feh, need to do another search-and-replace

Comment (jberkus) [reply releases/release-1.11/release-1.11.md]: will do after we get some other comments

**Comment by idvoretskyi** [comment]
@jberkus also, my suggestion is to submit dedicated PR's for a new release team role nomination and the draft schedule. 

While the draft schedule is currently under heavy discussion, we could have CJW nomination already discussed or merged :)

**Comment by jberkus** [comment]
@idvoretskyi 

Hmmm, good point, let me kill off this branch and recreate separate ones for team & schedule.

**Pull request closed without merging**